base/src/main/java/uk/ac/ebi/atlas/species/services/PopularSpeciesDao.java

```diff
@@ -21,7 +21,7 @@
             "count(experiment_organism.organism) AS c " +
             "FROM experiment " +
             "LEFT OUTER JOIN  experiment_organism ON experiment_organism.experiment=experiment.accession " +
-            "WHERE private='F' GROUP BY experiment.type, experiment_organism.organism";
+            "WHERE private=FALSE GROUP BY experiment.type, experiment_organism.organism";
```

Comment:
These changes will require updating every existing experiment in the staging/prod databases, I'm guessing. Something to keep in mind.